pyproject.toml

```diff
@@ -80,6 +81,8 @@ structlog = "*"
 tortoise-orm = "*"
 tox = "*"
 trio = "*"
+types-redis = "^4.3.21.2"
```

Comment:
so types should go into the `.pre-commit-config.yaml`- for mypy, refurbed (uses mypy) and pyright.

Reply:
Yeah, I even did this later on. .pre-commit-config.yaml has the right types. I just added this initially to make my local mypy happy. Will fix